test/dummy/app/components/header/header.js

```diff
@@ -1 +1 @@
-console && console.log('header')
+console && console.log('header');
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>'console' is not defined.